### PR TITLE
Add support for readonly containers

### LINKED ISSUE #203: Support --read-only root fs
Patch should be trivial (something in the line of adding `ReadonlyRootfs: d.Get("read_only").(bool),` to `HostConfig`).

I need this right now so will patch, but won't have time to add docs and tests right away.

### PR DESCRIPTION
**Pull request by dubo-dubon-duponey**
Fix #203 

Ping @mavogel 🎁;-)

### CODE REVIEW

**Comment by dubo-dubon-duponey** [comment]
Damn you travis.

@mavogel mind restarting the build? Maybe it's just transient.

**Comment by mavogel** [comment]
A restart of the build does not seem to solve it unfortunately @dubo-dubon-duponey

**Comment by dubo-dubon-duponey** [comment]
Have to look more into it. Now it's SIGSEGV...

**Comment by dubo-dubon-duponey** [comment]
@mavogel sorry, I don't get it...

The test runner (testing.tRunner.func1) itself is segfaulting (while it looks to me the test I'm introducing is very similar to an existing one).

Any pointers?

**Comment by mavogel** [comment]
No clue yet, but I'll try to take look later today

File: docker/resource_docker_container_test.go [hunk 1]
```diff
@@ -1100,6 +1100,33 @@ func TestAccDockerContainer_rm(t *testing.T) {
...
+		return nil
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDockerContainerReadOnlyConfig,
+				Check: resource.ComposeTestCheckFunc(
```
Comment (mavogel) [inline docker/resource_docker_container_test.go]: add `testAccContainerRunning("docker_container.foo", &c),` like in the other tests and it passes ;-) if not  `c`  will be `nil @dubo-dubon-duponey

Comment (dubo-dubon-duponey) [reply docker/resource_docker_container_test.go]: 🤦‍♂️
Hiding in shame -  I'll blame... a lapse in judgment :p

**Comment by dubo-dubon-duponey** [comment]
Should be good to go now!

Thanks a lot for the help @mavogel

File: docker/resource_docker_container_test.go (hunk 1 above)
Comment (mavogel) [reply docker/resource_docker_container_test.go]: No worries :) I am also in this club... sometimes hehe

**Pull request merged**